### PR TITLE
What's New in DevTools 123

### PR DESCRIPTION
**Pull request by leahmsft**
Rendered article for review:
* https://review.learn.microsoft.com/microsoft-edge/devtools-guide-chromium/whats-new/2024/03/devtools-123?branch=pr-en-us-3107

AB#49562355

### CODE REVIEW

**Comment by captainbrosset** [comment]
Let's remember to complete https://github.com/MicrosoftDocs/edge-developer/pull/3059 once this PR gets merged.

**Review by mikehoffms — APPROVED** [review]
Approved PR after pushing commits for a Writer/Editor pass.

File: microsoft-edge/devtools-guide-chromium/whats-new/2024/03/devtools-123.md
Comment (mikehoffms) [inline microsoft-edge/devtools-guide-chromium/whats-new/2024/03/devtools-123.md:102]: @leahmsft maybe remove some links

File: microsoft-edge/devtools-guide-chromium/whats-new/2024/03/devtools-123.md
Comment (mikehoffms) [inline microsoft-edge/devtools-guide-chromium/whats-new/2024/03/devtools-123.md:35]: link object elements
@leahmsft

Comment (leahmsft) [reply microsoft-edge/devtools-guide-chromium/whats-new/2024/03/devtools-123.md:35]: sorry i didn't make it clear that `(object elements)[]` is the name of the object. will fix in next commit

File: microsoft-edge/devtools-guide-chromium/whats-new/2024/03/devtools-123.md
Comment (mikehoffms) [inline microsoft-edge/devtools-guide-chromium/whats-new/2024/03/devtools-123.md:25]:
```suggestion
* An `Object` and its properties.
* An `Array` and its contents.
* A `Map` and its contents.
* A `Set` and its contents.
```
Confirmed that they need backticks:
* https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object
* https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array
* https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map
* https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set

Comment (mikehoffms) [reply microsoft-edge/devtools-guide-chromium/whats-new/2024/03/devtools-123.md]: added backticks

**Pull request merged**
